### ISSUE TITLE
Unexpected warning message in web_matrix_to_df function

### ISSUE THREAD

**Issue by valentinitnelav** [valentinitnelav/bootstrapnet#22]
I get the following warning message when I run `web_matrix_to_df`:

```r
library(bootstrapnet)
library(bipartite)

data(vazarr)

test <- bootstrapnet::web_matrix_to_df(web = vazarr)

# Warning message:
# In if (class(web) != "matrix") stop("`web` must be a matrix, e.g. `bipartite::Safariland`") :
#   the condition has length > 1 and only the first element will be used
```

**Comment by valentinitnelav** [comment]
This happens because `class` now returns two values: `[1] "matrix" "array"`. Then change condition in `web_matrix_to_df` to something like:

```r
if (! is.matrix(web)) stop("`web` must be a matrix, e.g. `bipartite::Safariland`")
```

**Comment by valentinitnelav** [comment]
Fixed with commit a119272b1373ac2c0a6bff30bf56cdab5ab8de9c
